docs/content/manual/harvester-rancher/4678-load-balancer-different-roles-pool.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Create load balancer on different roles pools in RKE2 cluster
+---
+
+* Related issues: [#4678](https://github.com/harvester/harvester/issues/4678) [BUG] Unable to successfully create LoadBalancer service from Guest cluster post 1.2.1 upgrade
+
+
+
+## Category: 
+* Rancher 
+
+## Verification Steps
+1. Prepare three nodes Harvester and import with Rancher
+1. Provision a RKE2 guest cluster in Rancher
+1. Create the first pool, specify the `etcd` and `control plane` roles
+1. Create the second pool, specify the `work` role only
+1. Add the following cloud config in the `User Data` of each pool
+    ```
+    write_files:
+    - encoding: b64
+    content: {harvester's kube config, the cluster namespace should be same as the pool you created (base64 enconded)}
```

Comment:
`enconded` -> `encoded`

Reply:
Thanks for the check.
Correct the repo in line 21.